packages/blocks/src/api/index.native.js

```diff
@@ -15,6 +15,8 @@ export {
 	registerBlockType,
 	setUnknownTypeHandlerName,
 	getUnknownTypeHandlerName,
+	setUnregisteredTypeHandlerName,
+	getUnregisteredTypeHandlerName,
```

Comment:
hmm, copy paste typo? This seems duplicated.

Reply:
Can you remove deprecated methods?

Reply:
Where's the duplication? 

I didn't remove the deprecated methods because I thought it will break the mobile tests 🤷‍♂️

Reply:
Oh no, need to get my eyes checked apparently. I missed the `s` vs `g` in the start of the method names 🤦‍♂️, ignore me :)

Reply:
Right, it's egg and chicken problem 😅

Reply:
Regarding breaking the mobile tests atm, it should be fine since we're planning to update the mobile ref with the proper methods, right?

Reply:
Agreed it avoids a new PR later to remove those.

Reply:
Actually, maybe it's better to leave them to avoid impacting other Gutenberg PRs in the meantime and I'll make sure that this is something that will be removed in the `4.2` functions removal.